index.js

```diff
@@ -1,10 +1,22 @@
-export default store => {
-  store.getters = new Proxy(store.getters, {
-    get(target, name) {
-      if (name in target) {
-        return target[name];
-      }
-      console.warn("A nonexistent getter has been called:", name);
-    }
-  });
+export default _options => {
+  const options = {
+    logger: console.warn,
+    silent: process.env.NODE_ENV === 'production' || false,
+    ...(_options || {}),
```

Comment:
i prefer this
```js
export default (_options = {}) => {
  const options = {
    // ...
    ..._options,
  }
}
```

Reply:
I thought using default parameters will throw an error on destructuring if `no-object` given, but that is my misunderstanding. sorry.